src/applications/mhv/secure-messaging/tests/containers/FolderThreadListView.unit.spec.jsx

```diff
@@ -97,19 +106,47 @@ describe('Folder Thread List View container', () => {
       },
     };
 
-    const setup = (state = initialState) => {
-      return renderWithStoreAndRouter(<FolderThreadListView />, {
-        initialState: state,
-        reducers: reducer,
-        path: Paths.DELETED,
-      });
-    };
-    setup();
+    const screen = setup(initialStateTrash, Paths.DELETED);
 
     await waitFor(() => {
       expect(global.document.title).to.equal(
         `Trash ${PageTitles.PAGE_TITLE_TAG}`,
       );
     });
+    const folderName = screen.getByRole('heading', { level: 1 });
+    expect(folderName).to.exist;
+    expect(folderName).to.have.text(DefaultFolders.DELETED.header);
+
+    const folderDescription = screen.getByTestId('folder-description');
+    expect(folderDescription).to.exist;
+    expect(folderDescription).to.have.text(DefaultFolders.DELETED.desc);
+    expect(screen.queryByText('Start a new message')).to.not.exist;
+  });
+
+  it(`verifies page title tag, folder name, folder description and "Start a new message" link should NOT exist for CUSTOM FOLDER`, async () => {
+    const initialStateDrafts = {
+      sm: {
+        messageDetails: { message: messageResponse },
+        folders: { folder: customFolder, folderList },
+      },
+    };
+
+    const screen = setup(
+      initialStateDrafts,
+      `/folders/${customFolder.folderId}`,
+    );
+
+    await waitFor(() => {
+      expect(global.document.title).to.equal(
+        `${customFolder.name} ${PageTitles.PAGE_TITLE_TAG}`,
+      );
+      const folderName = screen.getByRole('heading', { level: 1 });
+      expect(folderName).to.exist;
+      expect(folderName).to.have.text(customFolder.name);
```

Comment:
not critical, but 3 lines above can be optimized in 1 line like so
```suggestion
      expect(screen.getByText(customFolder.name, { selector: 'h1' })).to.exist;
```